controller/RestTestCenterUsers.php

```diff
@@ -157,7 +157,7 @@ public function post()
      *       description="Role id",
      *       required=true,
      *       @OA\Schema(
-     *           enum={"http://www.tao.lu/Ontologies/TAOProctor.rdf#TestCenterAdministratorRole", "http://www.tao.lu/Ontologies/TAOProctor.rdf#ProctorRole"},
+     *           enum={"http://www.tao.lu/Ontologies/TAOProctor.rdf#TestCenterAdministratorRole", "http://www.tao.lu/Ontologies/TAOProctor.rdf#ProctorRole", "http://www.tao.lu/Ontologies/generis.rdf#taoSyncManager"},
```

Comment:
It should get updated in `TestCenterService::unassignUser()` and in the `swagger.json` file as well.